pkg/registration/supply_chain_factory.go

```diff
@@ -110,17 +127,9 @@ func (f *SupplyChainFactory) buildQuotaSupplyBuilder() (*proto.TemplateDTO, erro
 		Commodity(cpuAllocationType, true).
 		Commodity(memAllocationType, true)
 
-	switch f.stitchingPropertyType {
-	case stitching.UUID:
-		vmQuotaExtLinkBuilder.
-			ProbeEntityPropertyDef(supplychain.SUPPLY_CHAIN_CONSTANT_UUID, "UUID of the Node").
-			ExternalEntityPropertyDef(supplychain.VM_UUID)
-	case stitching.IP:
-		vmQuotaExtLinkBuilder.
-			ProbeEntityPropertyDef(supplychain.SUPPLY_CHAIN_CONSTANT_IP_ADDRESS, "IP of the Node").
-			ExternalEntityPropertyDef(supplychain.VM_IP)
-	default:
-		return nil, fmt.Errorf("Stitching property type %s is not supported.", f.stitchingPropertyType)
+	err := f.addVMStitchingProperty(vmQuotaExtLinkBuilder)
```

Comment:
The Quota SE should also buy a `Cluster` commodity, which can confine it in the current kubernetes cluster.